models.py

```diff
@@ -1042,3 +1042,5 @@ def get_query(self):
 
     def get_count_query(self):
         return self.session.query(db.func.count('*')).filter(or_(self.model.sender_token == current_user.token, self.model.recipient_token == current_user.token)) # pylint: disable=no-member
+
```

Comment:
what is this supposed to do?

Reply:
not sure... i don't remember. I have a feeling that this code was placed after version